src/pages/OrganizationPage/OrganizationPage.tsx

```diff
@@ -0,0 +1,91 @@
+import { isUserAdmin } from 'lib/users';
+import sortBy from 'lodash/sortBy';
+import { useQuery } from 'react-query';
+import { useNavigate } from 'react-router';
+import { useParams } from 'react-router-dom';
+
+import { OrgLogoUpload, LoadingModal } from 'components';
+import { scrollToTop } from 'components/MainLayout/MainLayout';
+import useConnectedAddress from 'hooks/useConnectedAddress';
+import { CircleRow } from 'pages/CirclesPage/CirclesPage';
+import { paths } from 'routes/paths';
+import { AppLink, Box, Button, Flex, Text } from 'ui';
+import { SingleColumnLayout } from 'ui/layouts';
+
+import { getOrgData, QUERY_KEY_MY_ORGS } from './getOrgData';
+
+import type { Awaited } from 'types/shim';
+type QueryResult = Awaited<ReturnType<typeof getOrgData>>;
+
+export const OrganizationPage = () => {
+  const orgId = Number.parseInt(useParams().orgId ?? '-1');
+  const navigate = useNavigate();
+  const address = useConnectedAddress();
+  const query = useQuery(
+    [QUERY_KEY_MY_ORGS, orgId],
+    () => getOrgData(orgId, address as string),
+    {
+      enabled: !!address,
+      staleTime: Infinity,
+    }
+  );
+  const org = query.data?.organizations_by_pk;
+
+  const goToCircle = (id: number, path: string) => {
+    scrollToTop();
+    navigate(path);
+  };
+
+  if (query.isLoading || query.isIdle || query.isRefetching)
+    return <LoadingModal visible note="OrganizationPage" />;
+
+  if (!org) {
+    navigate(paths.circles);
```

Comment:
just noticed this -- you can't call this during rendering, it needs to be in e.g. `useEffect`